docs/reference/experimental/sync/dataset.md

```diff
@@ -4,16 +4,6 @@ Contained within this file are experimental interfaces for working with the Syna
 Client. Unless otherwise noted these interfaces are subject to change at any time. Use
 at your own risk.
```

Comment:
This should have been removed before once we shifted to using the tutorial for Dataset instead.